pkg/services/featuremgmt/registry.go

```diff
@@ -1775,6 +1775,14 @@ var (
 			HideFromDocs:      true,
 			Expression:        "true", // enabled by default
 		},
+		{
+			Name:         "dashboardImageSharing",
```

Comment:
```suggestion
			Name:         "sharingDashboardImage",
```
or `renderDashboardImage`.

Nit, I think it's nice to describe the feature toggle general area in the first word and then add details, it helps when looking for feature toggles and debugging.